src/main/java/frc/robot/subsystems/LEDs.java

```diff
@@ -0,0 +1,84 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+import edu.wpi.first.wpilibj.LEDPattern;
+import edu.wpi.first.wpilibj.util.Color;
+import java.util.Map;
+
+public class LEDs extends SubsystemBase {
+    
+    private final AddressableLED ledStrip;
+    private final AddressableLEDBuffer ledBuffer;
+    private LEDPattern pattern; 
+
+    public LEDs(int port, int leds) {
+
+        this.ledStrip = new AddressableLED(port);
+        this.ledBuffer = new AddressableLEDBuffer(leds);
+
+        this.ledStrip.setLength(leds);
+        this.ledStrip.start();
+
+        setRainbowPattern();
+    }
+
+    @Override
+    public void periodic() {
+        // This method will be called once per scheduler run
+       // LEDPattern pattern = LEDPattern.progressMaskLayer(() -> m_elevator.getHeight() / m_elevator.getMaxHeight());
+       // pattern.applyTo(ledBuffer);
+
+        // Set the LEDs
+        ledStrip.setData(ledBuffer);
+    }
+
+    public void setSolidPatternC1() {
```

Comment:
You should also come up with better names for the functions that is more clear as to what it does.